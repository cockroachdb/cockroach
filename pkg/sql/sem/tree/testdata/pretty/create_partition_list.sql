CREATE TABLE students_by_list (
    id SERIAL,
    name STRING,
    email STRING,
    country STRING,
    expected_graduation_date DATE,   
    PRIMARY KEY (country, id))
    PARTITION BY LIST (country)
      (PARTITION north_america VALUES IN ('CA','US'),
      PARTITION australia VALUES IN ('AU','NZ'),
      PARTITION DEFAULT VALUES IN (default))
