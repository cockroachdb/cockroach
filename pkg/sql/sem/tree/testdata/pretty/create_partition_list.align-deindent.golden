// Code generated by TestPretty. DO NOT EDIT.
// GENERATED FILE DO NOT EDIT
1:
-
CREATE TABLE students_by_list (
	id
		SERIAL,
	name
		STRING,
	email
		STRING,
	country
		STRING,
	expected_graduation_date
		DATE,
	PRIMARY KEY (country, id)
)
PARTITION BY LIST (
	country
) (
	PARTITION north_america VALUES IN (
		'CA',
		'US'
	),
	PARTITION australia VALUES IN (
		'AU',
		'NZ'
	),
	PARTITION default VALUES IN (
		DEFAULT
	)
)

14:
--------------
CREATE TABLE students_by_list (
	id SERIAL,
	name
		STRING,
	email
		STRING,
	country
		STRING,
	expected_graduation_date
		DATE,
	PRIMARY KEY (country, id)
)
PARTITION BY LIST (
	country
) (
	PARTITION north_america VALUES IN (
		'CA',
		'US'
	),
	PARTITION australia VALUES IN (
		'AU',
		'NZ'
	),
	PARTITION default VALUES IN (
		DEFAULT
	)
)

16:
----------------
CREATE TABLE students_by_list (
	id SERIAL,
	name STRING,
	email
		STRING,
	country
		STRING,
	expected_graduation_date
		DATE,
	PRIMARY KEY (country, id)
)
PARTITION BY LIST (
	country
) (
	PARTITION north_america VALUES IN (
		'CA',
		'US'
	),
	PARTITION australia VALUES IN (
		'AU',
		'NZ'
	),
	PARTITION default VALUES IN (
		DEFAULT
	)
)

17:
-----------------
CREATE TABLE students_by_list (
	id SERIAL,
	name STRING,
	email STRING,
	country
		STRING,
	expected_graduation_date
		DATE,
	PRIMARY KEY (country, id)
)
PARTITION BY LIST (
	country
) (
	PARTITION north_america VALUES IN (
		'CA',
		'US'
	),
	PARTITION australia VALUES IN (
		'AU',
		'NZ'
	),
	PARTITION default VALUES IN (
		DEFAULT
	)
)

19:
-------------------
CREATE TABLE students_by_list (
	id SERIAL,
	name STRING,
	email STRING,
	country STRING,
	expected_graduation_date
		DATE,
	PRIMARY KEY (country, id)
)
PARTITION BY LIST (
	country
) (
	PARTITION north_america VALUES IN (
		'CA',
		'US'
	),
	PARTITION australia VALUES IN (
		'AU',
		'NZ'
	),
	PARTITION default VALUES IN (
		DEFAULT
	)
)

29:
-----------------------------
CREATE TABLE students_by_list (
	id SERIAL,
	name STRING,
	email STRING,
	country STRING,
	expected_graduation_date
		DATE,
	PRIMARY KEY (country, id)
)
PARTITION BY LIST (country) (
	PARTITION north_america VALUES IN (
		'CA',
		'US'
	),
	PARTITION australia VALUES IN (
		'AU',
		'NZ'
	),
	PARTITION default VALUES IN (
		DEFAULT
	)
)

34:
----------------------------------
CREATE TABLE students_by_list (
	id SERIAL,
	name STRING,
	email STRING,
	country STRING,
	expected_graduation_date DATE,
	PRIMARY KEY (country, id)
)
PARTITION BY LIST (country) (
	PARTITION north_america VALUES IN (
		'CA',
		'US'
	),
	PARTITION australia VALUES IN (
		'AU',
		'NZ'
	),
	PARTITION default VALUES IN (
		DEFAULT
	)
)

41:
-----------------------------------------
CREATE TABLE students_by_list (
	id SERIAL,
	name STRING,
	email STRING,
	country STRING,
	expected_graduation_date DATE,
	PRIMARY KEY (country, id)
)
PARTITION BY LIST (country) (
	PARTITION north_america VALUES IN (
		'CA',
		'US'
	),
	PARTITION australia VALUES IN (
		'AU',
		'NZ'
	),
	PARTITION default VALUES IN (DEFAULT)
)

47:
-----------------------------------------------
CREATE TABLE students_by_list (
	id SERIAL,
	name STRING,
	email STRING,
	country STRING,
	expected_graduation_date DATE,
	PRIMARY KEY (country, id)
)
PARTITION BY LIST (country) (
	PARTITION north_america VALUES IN (
		'CA',
		'US'
	),
	PARTITION australia VALUES IN ('AU', 'NZ'),
	PARTITION default VALUES IN (DEFAULT)
)

51:
---------------------------------------------------
CREATE TABLE students_by_list (
	id SERIAL,
	name STRING,
	email STRING,
	country STRING,
	expected_graduation_date DATE,
	PRIMARY KEY (country, id)
)
PARTITION BY LIST (country) (
	PARTITION north_america VALUES IN ('CA', 'US'),
	PARTITION australia VALUES IN ('AU', 'NZ'),
	PARTITION default VALUES IN (DEFAULT)
)

142:
----------------------------------------------------------------------------------------------------------------------------------------------
CREATE TABLE students_by_list (id SERIAL, name STRING, email STRING, country STRING, expected_graduation_date DATE, PRIMARY KEY (country, id))
PARTITION BY LIST (country) (
	PARTITION north_america VALUES IN ('CA', 'US'),
	PARTITION australia VALUES IN ('AU', 'NZ'),
	PARTITION default VALUES IN (DEFAULT)
)

159:
---------------------------------------------------------------------------------------------------------------------------------------------------------------
CREATE TABLE students_by_list (id SERIAL, name STRING, email STRING, country STRING, expected_graduation_date DATE, PRIMARY KEY (country, id))
PARTITION BY LIST (country) (PARTITION north_america VALUES IN ('CA', 'US'), PARTITION australia VALUES IN ('AU', 'NZ'), PARTITION default VALUES IN (DEFAULT))

302:
--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
CREATE TABLE students_by_list (id SERIAL, name STRING, email STRING, country STRING, expected_graduation_date DATE, PRIMARY KEY (country, id)) PARTITION BY LIST (country) (PARTITION north_america VALUES IN ('CA', 'US'), PARTITION australia VALUES IN ('AU', 'NZ'), PARTITION default VALUES IN (DEFAULT))


